RecoTracker/LSTCore/standalone/efficiency/python/lst_plot_performance.py

```diff
@@ -413,6 +414,18 @@ def get_chargestr(charge):
 def set_label(eff, output_name, raw_number):
     if "phi" in output_name:
         title = "#phi"
+    elif "_deltaEta" in output_name: # Added by Kasia
+        title = "#eta diffs"
+    elif "_deltaPhi" in output_name: # Added by Kasia
+        title = "#phi diffs"
```

Comment:
Can you remove the # Added by Kasia from this and the "Last six added by Kasia" above. The git diff will show authorship.